radiomics/base.py

```diff
@@ -63,13 +61,19 @@ def __init__(self, inputImage, inputMask, **kwargs):
       self.logger.warning('Missing input image or mask')
       return
 
+  def _initLesionWiseCalculation(self):
```

Comment:
Oh, I now see what you meant. Then perhaps you meant to call it `_initLesionBasedCalculation` as opposed to `_initPixelBasedCalculation`. I think we need a consult from a native speaker - calling for help @pieper!